api/models/rules.py

```diff
@@ -0,0 +1,28 @@
+"""Rule model."""
+from django.db import models
+from django.utils.translation import ugettext_lazy as _
+
+from .base import Base
+from .year import Year
+
+
+class Rules(Base):
+    """Stores rules, terms and conditions for year."""
+
+    year = models.ForeignKey(
+        Year,
+        verbose_name=_("Year"),
+    )
+    text = models.TextField(
```

Comment:
You can read manual to remember the Markdown features. It's easy to use, but extra responsibility to maintain.

Point of having it like this is to **limit** the admin from being distracted by extra features while he should be creating **text content**. We can also have TinyMCE with all features and all plugins for headings, tables, forms, iframes, flash movies and gif editors, but why do we need that? We don't. Should we pull in dependencies for something we don't need? We shouldn't.